source/mir/random/generic.d

```diff
@@ -0,0 +1,384 @@
+/**
+License: $(LINK2 http://boost.org/LICENSE_1_0.txt, Boost License 1.0).
+
+Authors: Sebastian Wilzbach, Ilya Yaroshenko
+
+TODO:
+- automatic method to identify inflection points?
+
+*/
+module mir.random.generic;
+
+alias f_t = double;
+
+import std.stdio;
```

Comment:
it's very convenient to have this in there for debugging - I can just insert writeln calls anymore. If you don't mind I will leave this in for a bit?